angular/src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,29 @@
+<nav>
+    <div class="nav-wrapper" >
+        <a href="index" class="brand-logo"><img id="logo" src="./assets/images/bttcircle.png" alt="logo"></a>
```

Comment:
This image's `alt` text should be more descriptive of its content, not just of its role, for example "BTT logo" or just "BTT" (I can't find the exact case for this one but it looks like an [informative image](https://www.w3.org/WAI/tutorials/images/informative/) to me x) ).